cfgov/v1/tests/test_wagtail_hooks.py

```diff
@@ -164,3 +165,29 @@ def test_multiple_sites_still_returns_relative_link(self):
             is_default_site=False
         )
         self.check_page_renders_as_relative_link()
+
+
+class TestServeLatestDraftPage(TestCase):
+    def setUp(self):
```

Comment:
Could I humbly suggest these alternative tests as a slightly clearer and less dependent on mocking?

```py
class TestServeLatestDraftPage(TestCase):
    def setUp(self):
        self.default_site = Site.objects.get(is_default_site=True)
        self.page = SimplePage(title='live', slug='test', content='test')
        self.default_site.root_page.add_child(instance=self.page)
        self.page.title = 'draft'
        self.page.save_revision()

    @override_settings(SERVE_LATEST_DRAFT_PAGES=[])
    def test_not_serving_draft_serves_published_revision(self):
        response = self.client.get('/test/')
        self.assertContains(response, 'live')
        self.assertIsNone(response.get('Serving-Wagtail-Draft'))

    def test_serving_draft_serves_latest_revision_and_adds_header(self):
        with override_settings(SERVE_LATEST_DRAFT_PAGES=[self.page.pk]):
            response = self.client.get('/test/')
            self.assertContains(response, 'draft')
            self.assertEqual(response['Serving-Wagtail-Draft'], '1')
```

Reply:
Done!

Reply:
It does give me pause to add an expose-draft option to all CFGOVPage-based pages.